### DISCUSSION TITLE
IMU providing upside-down gravity in stationary conditions

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#248] [Show and tell]
I came across a nice post on this argument I want to share and keep track of.

Read the post here: http://www.lunar.org/docs/LUNARclips/v5/v5n1/Accelerometers.html

In a nutshell, it holds that:
>An accelerometer is a device that senses deviations from free fall.

**Comment by francesco-romano** [comment]
@S-Dafarra @aikolina
